astrodendro/tests/test_io.py

```diff
@@ -111,6 +113,29 @@ def test_fits_auto(self):
         # recognize from signature
         d2 = Dendrogram.load_from('astrodendro-test')
 
+    def test_hdf5_with_wcs(self):
+        self.test_filename = 'astrodendro-test-wcs.hdf5'
+        test_wcs = WCS(header=dict(cdelt1=1, crval1=0, crpix1=1,
+                                 cdelt2=2, crval2=0, crpix2=1,
```

Comment:
oops! fixing it...
(@ChrisBeaumont Do you know how to get SublimeText to properly indent such sitations in python, automatically? I'm aware of plugins that will auto-format a whole file to pep8, but I find those too aggressive - usually I just want to be able to hit "tab" and have Sublime do the right thing. Emacs is better at this)